src/client/power-pages/actions-hub/Constants.ts

```diff
@@ -25,9 +25,30 @@ export const Constants = {
         OTHER_SITES: vscode.l10n.t("Other Sites"),
         ACTIVE_SITES: vscode.l10n.t("Active Sites"),
         INACTIVE_SITES: vscode.l10n.t("Inactive Sites"),
-        NO_SITES_FOUND: vscode.l10n.t("No sites found")
+        NO_SITES_FOUND: vscode.l10n.t("No sites found"),
+        NO_ENVIRONMENTS_FOUND: vscode.l10n.t("No environments found")
     },
     EventNames: {
-        ACTIONS_HUB_INITIALIZED: "actionsHubInitialized"
+        ACTIONS_HUB_INITIALIZED: "actionsHubInitialized",
+        ACTIONS_HUB_CURRENT_ENV_FETCH_FAILED: "Actions Hub Current Environment Fetch Failed",
     }
 };
+
+export interface AuthInfo {
+    userType: string;
+    cloud: string;
+    tenantId: string;
+    tenantCountry: string;
+    user: string;
+    entraIdObjectId: string;
+    puid: string;
+    userCountryRegion: string;
+    tokenExpires: string;
+    authority: string;
+    environmentGeo: string;
+    environmentId: string;
+    environmentType: string;
+    organizationId: string;
+    organizationUniqueName: string;
+    organizationFriendlyName: string;
+}
```

Comment:
I think it's better to move this next to `AuthManager.ts` since it's applicable at Power Platform level. What do you think?